pysat/tests/test_instrument.py

```diff
@@ -483,54 +510,62 @@ def test_prev_load_empty_iteration(self):
         return
 
     def test_next_fname_load_default(self):
-        """Test next day is being loaded (checking object date)."""
+        """Test next day loads (checking object date)."""
+
         self.ref_time = dt.datetime(2008, 1, 2)
         self.testInst.load(fname=self.testInst.files[0])
         self.testInst.next()
         self.out = self.testInst.index[0]
         assert (self.out == self.ref_time)
         self.out = dt.datetime(self.out.year, self.out.month, self.out.day)
         assert (self.out == self.testInst.date)
+        return
 
     def test_prev_fname_load_default(self):
-        """Test prev day is loaded when invoking .prev."""
+        """Test prev day loads when invoking `.prev()`."""
```

Comment:
```suggestion
        """Test prev day loads when invoking `.prev()` starting with filename."""
```

Comment is probably too long, but the main difference between this and a previous test is loading by filename to start.

Reply:
Is this a candidate for parametrize?

Reply:
Going to kick updating `test_prev_filename_load_default` and `test_prev_fname_load_default` with parametrize down the road.  Do both of these have the same docstring?

Reply:
The load calls involve different keyword arguments, in one case standard argument. So, probably.